pyradiosky/skymodel.py

```diff
@@ -545,72 +625,48 @@ def __init__(
                 if self.hpx_order is None:
                     self.hpx_order = "ring"
 
-                self.Ncomponents = self.hpx_inds.size
-                if ra is not None and dec is not None:
+                if self.frame is None:
                     warnings.warn(
-                        "Input ra and dec parameters are being used instead of "
-                        "the default healpix coordinates. These coordinates will "
-                        "not necessarily line up with healpix pixel indicies. "
-                        "If you are intentionally trying to overwrite healpix "
-                        "coordinates please ensure the pixel indices are properly "
-                        "updated."
+                        "In version 0.2.0, the frame keyword will be required for HEALPix maps. "
+                        "Defaulting to ICRS",
```

Comment:
I think we should make this change in v0.3.0 not v0.2.0 since we're currently on v0.1.2.

Reply:
This is fun. I can't find the test that covers this line :laughing:

Reply:
turns out it was being hidden by a `pytest.warns` call. Since it was the same _type_ as what was being compared against it thought everything was good. I changed this to use `uvtest.check_warnings` since that one explicitly checks the number of warnings too.